internal/services/mssql/mssql_database_resource_test.go

```diff
@@ -1226,6 +1248,32 @@ resource "azurerm_mssql_database" "test" {
 `, r.template(data), data.RandomInteger)
 }
 
+func (r MsSqlDatabaseResource) gpWithLicenseType(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "azurerm_mssql_database" "test" {
+  name         = "acctest-db-%[2]d"
+  server_id    = azurerm_mssql_server.test.id
+  sku_name     = "GP_Gen5_2"
+  license_type = "LicenseIncluded"
+}
+`, r.template(data), data.RandomInteger)
+}
+
+func (r MsSqlDatabaseResource) gpServerlessWithNullLicenseType(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "azurerm_mssql_database" "test" {
+  name         = "acctest-db-%[2]d"
+  server_id    = azurerm_mssql_server.test.id
+  sku_name     = "GP_S_Gen5_2"
+  license_type = null
```

Comment:
Omitting this is equivalent to `license_type = null`. Could you please remove this instead of explicitly setting to null
```suggestion
```

Reply:
Removed it